CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 2.4.0-dev
```

Comment:
Want to go right to a stable release? We don't have any other changes I know of coming.

Reply:
Ok with me.